spec/helpers-spec.coffee

```diff
@@ -36,3 +36,16 @@ describe "The Linter Validation Helper", ->
       lint: ->
     }
     expect(Helpers.validateLinter(linter)).toEqual(true)
+
+describe "The Command Execution Helper", ->
+  beforeEach ->
+    waitsForPromise ->
+      @command = Helpers.exec('echo')
+  it "should throw when no parameters are passed.", ->
+    expect( -> Helpers.exec()).toThrow()
+
+describe "The Command and FilePath Helper", ->
+  it "should throw when no parametes are passed.", ->
+    expect( -> Helpers.execFilePath()).toThrow()
+  it "should throw when no File Path is passed.", ->
+    expect( -> Helpers.execFilePath('echo')).toThrow()
```

Comment:
Can you do some example `exec` with the echo command?
and can you also add tests for parse by having some example input and a regex and matching their values?